bin/osg-authfile

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3.6
```

Comment:
Have you tried this? Does using the system python work with packages installed inside the virtualenv?

Reply:
Yeah, this is how I run things locally.  However, given we do `venv`-based installs, I agree that an absolute path isn't the best.  Is `/usr/bin/env python3` more kosher?

Reply:
Yep, that's what the other scripts do.

Reply:
Done in the latest update.